semantic/io.cpp

```diff
@@ -343,6 +343,8 @@ void parse(TrafficSign &trafficSign, const Json::Value &value)
     parse(trafficSign.views, Json::check(value, "views", Json::arrayValue));
     parse(trafficSign.size, Json::check(value, "size", Json::arrayValue));
     Json::get(trafficSign.className, value, "className");
+    Json::get(trafficSign.content, value, "content");
+    Json::get(trafficSign.classId, value, "classId");
```

Comment:
Do not include changes from Tomas, you only want to add your attribute.

Reply:
Resolved after merging changes from Tomas.